wiser/eval/util.py

```diff
@@ -341,3 +342,32 @@ def _get_p_r_f1(tp, fp, fn):
     f1 = round(2 * p * r / (p + r) if p > 0 or r > 0 else 0.0, ndigits=4)
 
     return p, r, f1
+
+    
+def tagging_rule_errors(instances, rule, error_type='fn', gold_label_key='tags', mode = 'span'):
```

Comment:
Is there a reason you put `gold_label_key` as the function parameter, but not 'WISER_LABELS'? Just wondering about consistency here because you treat 'WISER_LABELS' as a magic string in line 355.

Would it be better if you have another function parameter for it such as`wiser_label_key = "WISER_LABELS"`?